cltk/stop/middle_high_german/stops.py

```diff
@@ -0,0 +1,308 @@
+"""
+List generated by me, using TFIDF method. Note that there is no distinction between verbal froms or grammatical cases 
```

Comment:
TF-idf is a great place to start.

The stops list generally looks good but a few changes: 1) Please remove all nouns (eg, `herren`) and adjectives (if any). 2) Also remove any verbs except "auxiliary verbs" ([examples in MG here](https://www.dartmouth.edu/~deutsch/Grammatik/Verbs/modals.html)).

3) Also, you need to include (if you have not) all of the cases/numbers for pronouns (if you have not). For example, in MG, not just`ich` but also `mich`, `mir, `mein`.

You may need a grammar to help with this last step after tf-idf.

I don't know MHG so I will rely on you to help if anything I say here is incorrect.

Reply:
Thanks for the feedback! I will work on it :)

Reply:
@Sedictious You may also be interested in the CLTK ``CorpusStoplist`` class I have under development. : https://github.com/diyclassics/cltk/blob/stops-dev/cltk/stop/stop.py. Docs here: https://github.com/diyclassics/cltk/blob/stops-dev/docs/multilingual.rst#stoplist-construction. In fact, feedback on this new class would be helpful. Fwiw I plan on adding td-idf as a ``basis`` this weekend—I'll let you know when it is ready. PJB

Reply:
Thanks @diyclassics ! In fact, I checked your class just this morning, really clean and readable code. Will look into it on the weekend and let you know if I have any feedback. Btw, do you still have problem with dependencies? By the looks of it, replacing np and sci-kit with Standard Library wouldn't be impossible.